scripts/asm-installer/install_asm

```diff
@@ -259,11 +260,14 @@ OPTIONS:
                                       install. Specifying the CA is required
                                       for --mode migrate.  Allowed values for
                                       <CA> are {citadel|mesh_ca}.
-  -o|--operator_overlay  <FILE PATH>  The location of a YAML file to overlay on
-                                      the ASM IstioOperator. This option can be
-                                      omitted if not installing optional
-                                      features. To add multiple files, specify
-                                      them with multiple options one at a time.
+  -o|--option            <FILE NAME>  The name of a YAML file in the kpt pkg to
```

Comment:
would it makes sense to add the absolute url here to facilitate users interested in exploring this directory?

Reply:
Good idea, added